src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -66,7 +106,9 @@ Your website can check if your:
 
 Your website can check if your Android app is installed.
 
-* Android: Chrome 80 or later
+{% Compare 'better', 'Supported on' %}
+Android: Chrome 80 or later
+{% endCompare %}
```

Comment:
@robdodson @kaycebasques Is this a correct use of the macro?

I would avoid an improper use a macro. If an automated update is ever deployed on the site's content, it could break the intent of this formatting and no one would catch it for weeks or months.

Reply:
I'd argue it's a reasonable use for this. The compare widget is what developers should do, and this clearly calls out that they should test these types of apps.

Reply:
I think it's probably fine. Joe makes a good point that when we work on the CSS for the site we tend to think of the compare widget as a pair of do/don'ts so there might be _some_ risk that someday we'd make a change that could break this use case, but I don't think we have any immediate plans to radically alter the way this widget works.

So I guess my answer is something like "sure, but to joe's point note that there is a bit of incurred risk".

Reply:
Thanks - I've filed #4645 to track a new feature request for a 'Supported On' widget, and also included this bug in there so that if/when we get it, we know to come back and update this doc.